amf-client/shared/src/test/resources/org/raml/api/v10/resource-type-optional/model.json

```diff
@@ -16,7 +16,8 @@ resourceTypes:
         application/json:
           type: <<resourcePathName>>
 types:
-  implemented-post: {}
+  implemented-post:
```

Comment:
Default type "string"

Reply:
Idem that org/raml/api/v10/full/model.json, no type defined at source input.raml file.